readthedocs/projects/tasks.py

```diff
@@ -901,6 +901,23 @@ def setup_vcs(self, environment):
         """
         environment.update_build(state=BUILD_STATE_CLONING)
 
+        # Install a newer version of ca-certificates packages because it's
+        # required for Let's Encrypt certificates
+        # https://github.com/readthedocs/readthedocs.org/issues/8555
+        # https://community.letsencrypt.org/t/openssl-client-compatibility-changes-for-let-s-encrypt-certificates/143816
+        # TODO: remove this when a newer version of ``ca-certificates`` gets
+        # pre-installed in the Docker images
+        self.setup_env.run(
+            'apt-get', 'update', '--assume-yes', '--quiet',
+            user=settings.RTD_DOCKER_SUPER_USER,
+            record=False,
+        )
+        self.setup_env.run(
+            'apt-get', 'install', '--assume-yes', '--quiet', 'ca-certificates',
+            user=settings.RTD_DOCKER_SUPER_USER,
+            record=False,
+        )
+
```

Comment:
This seems a little scary to introduce to every build. Can we at least put it behind a feature flag that we default True or something, so we can easily turn it on/off in prod if we hit issues?

Reply:
Done!